.env

```diff
@@ -49,7 +49,6 @@ PROVIDER_UC_DATAAPP_URI=http://uc-dataapp-provider:9555
 CONSUMER_PORT=8091
 # mixed | form | http-header
 CONSUMER_MULTIPART_EDGE=mixed
-CONSUMER_MULTIPART_ECC=mixed
 CONSUMER_DATA_APP_ENDPOINT=https://be-dataapp-consumer:8083/data 
 CONSUMER_IDSCP2=true
```

Comment:
probably at the top (like "global" config) we can put a new variable IDSCP2 that sets up CONSUMER_IDSCP2 and PROVIDER_IDSCP2 to the same value since it makes no sense to have it enabled on one side and not on the other. It should also raise an error during the process